igvc_description/urdf/jessi.urdf

```diff
@@ -42,7 +42,7 @@
     <inertial>
       <origin xyz="0 0 0" />
       <mass value="30.0" />
-      <inertia  ixx="0.994968" ixy="-0.100194"  ixz="0.0"  iyy="0.994968"  iyz="0.0"  izz="-1.0" />
+      <inertia  ixx="0.094969" ixy="-0.100194"  ixz="0.0"  iyy="0.094968"  iyz="0.0"  izz="-0.1" />
```

Comment:
We should calculate these from the CAD for realsies.

Reply:
Reverted changes.